third_party/tlx/tutorials/hopper-fa-ws-pipelined-pingpong_test.py

```diff
@@ -131,19 +131,21 @@ def _attn_fwd_ws_pipelined_pingpong(sm_scale, M,  #
             k_tile = tlx.local_trans(k_tiles[k_buf_id])
 
             if cid == 0:
-                # Consumer 0 waits for Consumer 1 to reach synchronization point at barrier 9.
-                tlx.named_barrier_wait(9, 256)
+                # Consumer 0 waits for Consumer 1 to reach synchronization point at barrier 7.
+                tlx.named_barrier_wait(7, 256)
             else:
-                # Consumer 1 signals its arrival at barrier 9.
+                # Consumer 1 signals its arrival at barrier 7
```

Comment:
Is it possible to reuse 9 and 10 here, as the qk dot is basically peeled from the first loop iteration?